hr_skill/views/hr_skill.xml

```diff
@@ -35,7 +35,7 @@
     <!-- Menu and Actions -->
 
     <menuitem name="Skills Management"
-              parent="hr.menu_hr_configuration"
+              parent="hr.menu_human_resources_configuration"
               id="menu_hr_skill_mgt"/>
 
```

Comment:
@nicomacr Can you update the menus, i don't think the Skill Structure is needed, since you have the childs on each skill, the name is computed from all parents, also you can remove the Skill Management parent menu, only to leave the Skills menu in the configuration...